### ISSUE TITLE
1.0.6 update error on startup

### ISSUE THREAD

**Issue by steveheinsch** [SSMCD/bluetti2mqtt#7]
I just upgraded from 1.0.5 and restarted HA. I noticed the plugin wasn't running so hit restart and got a popup

```
Go to configuration

does not match regular expression ^([0-9A-Fa-f]{2}[:]){5}([0-9A-Fa-f]{2})$.

[cancel] [edit config]
```

my config hasn't changed

```
mqtt_username: [user]
mqtt_password: [pass]
mqtt_host: 192.168.0.x
mqtt_port: 1883
bt_mac: [device 1 mac] [device 2 mac]
poll_sec: 5
ha_config: advanced
scan: true
debug: false
log: false
```

If I choose edit config, it looks normal, and then click save I get the same error as above except it also spits out a json object with my config options.

**Comment by SSMCD** [comment]
This should be fixed in 1.0.7

**Comment by steveheinsch** [comment]
Fixed, thank you
